contrib/coredns/policy/policy.go

```diff
@@ -116,12 +128,17 @@ func (p *PolicyMiddleware) getEDNS0Attrs(r *dns.Msg) ([]*pb.Attribute, bool) {
 						ip := net.IP(e.Data)
 						value = ip.String()
 					}
+					from := m.stringOffset
+					to := m.stringOffset + m.stringSize
```

Comment:
If Offset and Size are not specified, they are both "0", isn't that an issue - or did I miss something?